packages/test/src/index.ts

```diff
@@ -18,8 +18,12 @@ import "./plugin"
 
 import * as test from "node:test"
 
+import * as chai from "chai"
+import chaiAsPromised from "chai-as-promised"
 import * as semver from "semver"
 
+chai.use(chaiAsPromised)
```

Comment:
Didn't end up needing it for changed tests in the end but it's a useful plugin to have around for testing whether promises resolve/reject etc